service.yml

```diff
@@ -0,0 +1,28 @@
+name: kafka-images
+lang: unknown
+lang_version: unknown
+git:
+  enable: true
+codeowners:
+  enable: true
+semaphore:
+  enable: true
+  pipeline_type: cp-dockerfile
+  docker_repos: [
+    'unsynced-confluentinc/cp-server-connect',
+    'unsynced-confluentinc/cp-server-connect-base',
+    'unsynced-confluentinc/cp-kafka-connect',
+    'unsynced-confluentinc/cp-kafka-connect-base',
+    'unsynced-confluentinc/cp-enterprise-kafka',
+    'unsynced-confluentinc/cp-kafka',
+    'unsynced-confluentinc/confluent-local',
+    'unsynced-confluentinc/cp-server',
+    'unsynced-confluentinc/cp-zookeeper'
+  ]                                                   # Required: Replaces dockerRepos: value should be the similar as dockerRepos in your Jenkinsfile. Replace confluentinc with unsynced-confluentinc
```

Comment:
don't recommend leaving these comments. This can change in the future and can be confusing if not up to date.